DESCRIPTION

```diff
@@ -24,12 +23,12 @@ Imports:
     scales,
     grDevices
 Remotes:
-    reconhub/incidence,
-    nickreich/coarseDataTools@hackout3
+    nickreich/coarseDataTools
```

Comment:
If we don't have one already, should we add an issue to remove this once this has been published to CRAN?

Reply:
:+1: We should add an issue.

Reply:
https://github.com/annecori/EpiEstim/issues/71